adapter/src/main/java/org/jvalue/ods/adapterservice/config/SwaggerConfig.java

```diff
@@ -0,0 +1,23 @@
+package org.jvalue.ods.adapterservice.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class SwaggerConfig {
+
+  @Bean
+  public Docket api() {
+    return new Docket(DocumentationType.SWAGGER_2)
```

Comment:
There is no support for OpenAPI3 yet, right?

Reply:
Actually, I think there is but it requires another library / mechanism. Probably makes sense to do so?

Reply:
I think it make sense to use the newest version of the specification if it is included in a spring boot plugin. If we need some kind of workaround or a lot of effort I would rather wait until the spring boot compatible version is updated to the new spec.